app/api/appointments/getAppointments.ts

```diff
@@ -45,34 +58,43 @@ export async function getAvailableTimeSlots(jobType: JobType): Promise<DateEntry
     //     pageSize: 1000,
     // });
     // const allTechs = getTechsList.data || getTechsList;
-    const techsSkillsList: TechnicianToSkillsMapping[]  = await getTechnicianToSkills();
+    // const techsSkillsList: TechnicianToSkills[]  = await getTechnicianToSkills();
+    logger.info(`${prompt} Invoking getAllTechnicians function...`);
+    const techsSkillsList: TechnicianToSkills[]  = await getAllTechnicians();
+    logger.info(`${prompt} Fetched ${techsSkillsList.length} technicians from getAllTechnicians function.`);
+
 
     // Filter to only enabled technicians
+    logger.info(`${prompt} Filtering just ENABLED technicians...`);
     const enabledTechsSkillsList = techsSkillsList.filter(tech => tech.enabled === true);
-    logger.info(`Found ${enabledTechsSkillsList.length} enabled technicians with skills: ${enabledTechsSkillsList.map(tech => tech.technicianName).join(', ')}`);
-
     if (enabledTechsSkillsList.length === 0) {
-      logger.warn("No enabled technicians available");
+      logger.warn(`${prompt} No ENABLED technicians available`);
       return [];
     }
+    logger.info(`${prompt} Found ${enabledTechsSkillsList.length} ENABLED technicians (${enabledTechsSkillsList.map(tech => tech.technicianName).join(', ')})`);
 
     // Get ST Job Type
+    logger.info(`${prompt} Hitting ServiceTitan API for jobType with serviceTitanId: ${jobType.serviceTitanId}`);
```

Comment:
This is good. I've almost never logged too little. And it's easy to "turn down" logging later. but you can't "turn up" logging once it's already happened.